qatrfm/config/default.tf

```diff
@@ -25,15 +25,15 @@ provider "libvirt" {
 
 resource "libvirt_volume" "myvdisk" {
   name = "qatrfm-vdisk-${var.basename}-${count.index}.qcow2"
-  count = "${var.count}"
+  count = "${var.num_domains}"
   pool = "default"
   source = "${var.image}"
   format = "qcow2"
 }
 
 resource "libvirt_network" "my_net" {
    name = "qatrfm-net-${var.basename}"
-   addresses = ["${var.network}"]
+   addresses = ["10.${var.net_octet}.0.0/24"]
```

Comment:
because I had some doubt I google this and generally looks like you can have IP in form x.y.z.0 ( ending on zero ) BUT "there are some device and some cases where this could be treated as `network IP`". because we don't any strong need to have exactly this IP may we change 0 to 1 ?

Reply:
I thought we specify the network here. And the exact IP which will be used for the VM is somehow calculated from terraform from that  pool. 
And terraform knows which pool to use from line: https://github.com/jlausuch/qatrfm-lib/pull/1/files#diff-5805ca2ddf719864e29bfbf90857b877R49

Reply:
ah ok , thanks for clarification